docs/source/tutorials/oregon.rst

```diff
@@ -0,0 +1,688 @@
+
+**Statistical Significance**: Both simple and ML-adjusted local estimators show similar patterns, providing robust evidence that Medicaid assignment has significant distributional effects on emergency department costs for compliers. The confidence intervals indicate that these effects are statistically significant across most cost levels.
+
+**Non-Compliance Considerations**: The LDTE analysis accounts for the fact that not all individuals assigned to treatment actually enrolled in Medicaid, providing a more accurate estimate of the effects for those who actually receive treatment when assigned.
+
+Cost Analysis with Local PTE
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    # Compute Local Probability Treatment Effects
+    lpte_simple, lpte_lower_simple, lpte_upper_simple = simple_local_estimator.predict_lpte(
+        target_treatment_arm=1,  # Z=1 Selected for treatment (Enrolled)
+        control_treatment_arm=0,  # Z=0 Not selected for treatment (Not enrolled)
+        locations=[-1] + outcome_ed_costs_locations
+    )
+
+    lpte_ml, lpte_lower_ml, lpte_upper_ml = ml_local_estimator.predict_lpte(
+        target_treatment_arm=1,  # Z=1 Selected for treatment (Enrolled)
+        control_treatment_arm=0,  # Z=0 Not selected for treatment (Not enrolled)
+        locations=[-1] + outcome_ed_costs_locations
+    )
+
+    fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(15, 6))
+
+    # Simple local estimator
+    plot(outcome_ed_costs_locations[1:], lpte_simple, lpte_lower_simple, lpte_upper_simple,
+        chart_type="bar",
+        title="Effects of Emergency Department Costs (Simple Local Estimator)",
+        xlabel="Emergency Department Costs", ylabel="Local Probability Treatment Effect",
+        color="purple",
+        ax=ax1)
+
+    # ML-adjusted local estimator
+    plot(outcome_ed_costs_locations[1:], lpte_ml, lpte_lower_ml, lpte_upper_ml,
+        chart_type="bar",
+        title="Effects of Emergency Department Costs (ML-Adjusted Local Estimator)",
+        xlabel="Emergency Department Costs", ylabel="Local Probability Treatment Effect",
+        ax=ax2)
+    plt.tight_layout()
+    plt.show()
+
+The Local Probability Treatment Effects analysis produces the following visualization:
+
+.. image:: ../_static/oregon_lpte_costs_comparison.png
+   :alt: Oregon Health Insurance Experiment LPTE Analysis
+   :width: 800px
+   :align: center
+
+The side-by-side bar charts show probability treatment effects across different emergency department cost intervals, revealing how Medicaid enrollment affects healthcare utilization patterns:
+
+**Cost Distribution Effects**: The LPTE analysis shows how Medicaid assignment changes the probability of compliers incurring emergency department costs in specific ranges. Positive bars indicate cost intervals where Medicaid assignment increases the likelihood of incurring costs in that range, while negative bars show intervals where it decreases the probability.
```

Comment:
This is a correct explanation about how to understand LPTE, but what's the insight specifically for this data?